microsoft/atlas-design
# Adds Aspect ratio atomic

Task: task-862482

Link: preview-556

When videos are added and the container is wider or taller than the aspect ratio of the video it creates black bars in the video player. These ratio atomics address this issue by allowing the player to scale according to the ratio when applied directly to the `iframe` or `video` element. It can also be used to crop images to a certain aspect ratio. This can also be applied to images but will stretch the img.

## Testing

1. Visit https://design.learn.microsoft.com/pulls/556/atomics/aspect-ratio.html
2. Open dev tools and verify classes are avialable.
   Expected result: Classes are avialable and scale element to different ratios.


## Contributor checklist
